docs/tutorial/3-build-your-first-dapp.md

```diff
@@ -5,9 +5,9 @@ slug: /build-your-first-dapp
 
 # Build Your First Dapp
 
-To get started with building a Decentralized Application (Dapp), clone the following repository and follow the README instructions: [https://github.com/paritytech/contracts-boilerplate](https://github.com/paritytech/contracts-boilerplate).
-
-The repository contains two boilerplates:
-
-1. **React and Ether.js**: This boilerplate uses React and Ether.js, a well-established library for interacting with Solidity contracts.
-2. **Modern TypeScript with wagmi**: This boilerplate offers a more modern approach, fully typed, and includes powerful React hooks provided by wagmi.
+To get started with building a Decentralized Application (Dapp), spin up a repository, using `create-polkadot-dapp`:
+```
+npx create-polkadot-dapp@latest
```

Comment:
Isn't that the same?

```suggestion
npx create-polkadot-dapp
```

Reply:
It forces downloading the latest version of `create-polkadot-dapp` and the templates.  
Otherwise, if you've tried create-polkadot-dapp before, it'll spin up the old version.